src/docker.py

```diff
@@ -51,6 +51,8 @@ def extract_resources_from_docker(self):
             self.__extract_from_docker('cloverio/clover-para', '/opt/clover/bin/clover', '/opt/specs')
         elif self.chain_name == 'polkadex':
             self.__extract_from_docker('polkadex/parachain', '/data/bin/parachain-polkadex-node', '/data/polkadot-parachain-raw.json')
+        elif self.chain_name.startswith('crust-'):
```

Comment:
Perhaps better, for the code reader, to do the `chain_name in ['crust-mainnet', 'crust-...']` variant?

From this one doesn't know what is a valid suffix following the `-`.

Also, should we allow just `crust` as an alias for mainnet?